### PR TITLE
Add plugins jobs

### PR DESCRIPTION
**Pull request by mstemm**
Add support for falcosecurity/plugins repo in prow/poiana.

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -209,7 +209,9 @@ branch-protection:
           branches:
             master:
               protect: true
-        template-repository:
+        plugins:
+          required_pull_request_reviews:
+            required_approving_review_count: 2
```
Comment (leogr) [inline config/config.yaml:214]:
```suggestion
```

Since it's an incubating project, we could stick with only 1 approval at the moment.

Comment (mstemm) [reply config/config.yaml]: Ok, fixed.

Comment (leogr) [reply config/config.yaml]: Thank you!

**Review by leogr — APPROVED** [review]
/approve

**Pull request merged**